sonic_platform_base/sonic_xcvr/api/public/cmisTargetFWUpgrade.py

```diff
@@ -97,6 +120,31 @@ def get_transceiver_info_firmware_versions(self):
         self.set_firmware_download_target_end(TARGET_E0_VALUE)
         return return_dict
 
+    def _is_remote_target_accessible(self):
+        """
+        Once the target is changed to remote, any register apart from the TARGET_MODE register
+        will have the value 0xff if the remote target is powered down.
+        Assumption:
+            The target mode has already been set to the desired remote target.
+        Returns:
+            True if the remote target is accessible from the local target, False otherwise.
+        """
+        module_type = self.get_module_type()
+        if 'Unknown' in module_type:
+            return False
+
+        return True
+
+    def _handle_error_and_restore_target_to_E0(self, error_message):
```

Comment:
@mihirpat1 this API be rename to _restore_target_E0(). Let the caller print the log errors before calling this. This allows reusing this API in future without throwing error logs

Reply:
@prgeor I have addressed this now.